scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -k <S3 key prefix>"
```

Comment:
Should we somehow indicate in these usage-strings which flags are optional, and which are compulsory?

(Traditionally in unix-cli-land, it's only the optional arguments which have flags, and all compulsory arguments are specified by position. But obviously with these scripts having so many arguments, having compulsory positional-only arguments would be a bit unwieldy!)